test/TestProjects/MgmtMockTest/Samples/Generated/DiskEncryptionSets_CreateOrUpdate$$Create a disk encryption set with key vault from a different subscription..cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+#region Usings
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Models;
+using Azure.ResourceManager.Resources;
+using MgmtMockTest;
+using MgmtMockTest.Models;
+#endregion
+
+#region Parameter decalarations
+// Below parameters are extracted from swagger example files, please use your own values to execute the API lively.
+ArmClient GetArmClient() => new ArmClient(new DefaultAzureCredential());
+var subscriptionId = Environment.GetEnvironmentVariable("SUBSCRIPTION_ID") ?? "00000000-0000-0000-0000-000000000000";
```

Comment:
Since we have the comment "Below parameters are extracted from swagger example files, please use your own values to execute the API lively." we really do not need to put the extra GetEnvironmentVariable thing, we could just 
```suggestion
var subscriptionId = "00000000-0000-0000-0000-000000000000";
```
and expect our customer reads the comment above and change it to their own subscriptionId, either hard code here or getting it from environment variables.
This also simplify our generator a few